common-util/Login/SolanaWallet.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jest/require-hook */
+import {
+  WalletModalProvider as ReactUIWalletModalProvider,
+  // WalletConnectButton,
+  // WalletDisconnectButton,
+  // WalletModalButton,
+  WalletMultiButton,
+} from '@solana/wallet-adapter-react-ui';
+
+require('@solana/wallet-adapter-react-ui/styles.css');
+
+export const SolanaWallet = () => (
+  <ReactUIWalletModalProvider>
```

Comment:
We're almost on the same page here but we can have the WalletProvider in this component, too
<img width="486" alt="image" src="https://github.com/valory-xyz/autonolas-registry-frontend/assets/66292936/2ee9979f-ca13-4020-880d-6a6f208c99e4">

Reply:
Ah yes, I see – good point